alert/mail.py

```diff
@@ -2,11 +2,18 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-import boto
+import boto.ses
 
 from email.mime.application import MIMEApplication
 from email.mime.multipart import MIMEMultipart
 from email.mime.text import MIMEText
+import json
+
+with open("aws-settings.json", "r") as f:
+  settings = json.load(f)
+  AWS_ACCESS_KEY = settings["AWS_ACCESS_KEY"]
+  AWS_SECRET_ACCESS_KEY = settings["AWS_SECRET_ACCESS_KEY"]
+  AWS_REGION = settings["AWS_REGION"]
 
```

Comment:
There is no need for this as boto picks up the credentials from ~/.aws/credentials.